js/templates/site.es6.js

```diff
@@ -1,19 +1,21 @@
 const bel = require('bel');
 
 module.exports = function () {
-    
+
+    var domain = this.model.domain;
+
     return bel`<div class="js-site js-menu-section">
         <ul class="js-menu-item-list">
             <li class="js-site-item">
-                <span class="js-site-domain">${this.model.domain}</span>
+                <span class="js-site-domain">${domain}</span>
                 <div class="js-site-whitelist-toggle-bg js-site-whitelist-${this.model.isWhitelisted}"><div class="js-site-whitelist-toggle-fg js-site-whitelist-fg-${this.model.isWhitelisted}"></div></div>
```

Comment:
Just a style thing in the templates - would be nice to always break at a reasonable width on these files. Not everybody is going to have their editor set as wide as you and it makes edits annoying for other ppl.